test/unit/auth/project-config.spec.ts

```diff
@@ -51,6 +51,18 @@ describe('ProjectConfig', () => {
         },
       ],
     },
+    recaptchaConfig: {
```

Comment:
Do we need this? 
`updateProjectConfigRequest3` already has the same update config for recaptcha and I believe we are testing that functionality?

Reply:
Yes, this is also to fix the unit test failure.